week-2/mandatory/2-ecommerce-db/task.md

```diff
@@ -35,16 +35,61 @@ Open the file `cyf_ecommerce.sql` in VSCode and examine the SQL code. Take a pie
 Once you understand the database that you are going to work with, solve the following challenge by writing SQL queries using everything you learned about SQL:
 
 1. Retrieve all the customers' names and addresses who live in the United States
+
+SELECT name, address
+  FROM customers
+  WHERE country = 'United States';
+
 2. Retrieve all the customers in ascending name sequence
+
+SELECT *
+  FROM customers
+  ORDER BY name ASC;
+
 3. Retrieve all the products whose name contains the word `socks`
+SELECT *
+
+  FROM products
+  WHERE product_name ILIKE '%socks%';
```

Comment:
This would be a case insensitive statement; e.g. `socks` and `Socks` and `SoCKs` would all meet this. The question is a little ambiguous as to whether this should be the case, but just something to be aware of.